README.md

```diff
@@ -173,6 +174,16 @@ Kind: *Trusted Indication*.
 
 Errors:
 
+### `AccountClaim`
+Sends a claim verified to be from the `source` to the `destination` on the parachain. It has an `expiration` timestamp and a `scope` used to define the scope under which such a `claim` is made. A chain should only send such a claim if it is verified to have been intended to be sent by the `source`. It facilitates authentication across parachains.
+
+- `source: MultiLocation`
+- `destination: MultiLocation`
+- `expiration: Compact<u48>`: Timestamp of the claim expiration
```

Comment:
when exactly would this be used?

Reply:
Any certificate or claim should have an expiration IMO. Just looking at how signed statements would be used when working in different contexts. For aanything from SSL to meta-transactions in crypto, most often you do need some sort of expiry or nonce but nonces wouldn't make sense in this instance.

Reply:
@gavofyork are you ok with leaving this in?